src/components/test-summary/test-summary-section/test-summary-section.html

```diff
@@ -1,10 +1,11 @@
-<ion-card class="mes-test-summary-section-card" [ngClass]="{'hidden': summary.total === 0}">
-    <ion-card-header margin-bottom [color]="getFaultTitleColour(summary.title)">
-        {{summary.title}} - {{summary.total}}
-    </ion-card-header>
-    <ion-card-content>
-        <ul class="mes-test-summary-list">
-            <li *ngFor="let fault of summary.faults">{{fault.name}} <span *ngIf="fault.total > 1">({{fault.total}})</span></li>
+<ion-row>
+    <ion-col col-4>
+        <div id="summary-total">{{summary.total}}</div>
```

Comment:
Is this section component to be used multiple times on the same page? If so, `class` probably better than `id`. I'm not sure of the convention, but I don't see any per-instance ID prefixing in the DOM.